redis/tests/test_sentinel.rs

```diff
@@ -106,20 +116,31 @@ fn test_sentinel_server_down() {
 
     let sentinel = context.sentinel_mut();
 
-    for _ in 0..20 {
-        let mut replica_con = sentinel
-            .replica_rotate_for("master1")
-            .unwrap()
-            .get_connection()
-            .unwrap();
+    let mut replica_conn_infos = vec![];
+
+    for _ in 0..3 {
```

Comment:
is this a verbatim copy of lines 85 and on? if so, please extract to a function

Reply:
Yes, and there is something similar in the async tests, I'll extract both to a function.